fractal_server/app/api/v1/_aux_functions.py

```diff
@@ -210,3 +212,36 @@ async def _get_job_check_owner(
             detail=f"Invalid {project_id=} for {job_id=}",
         )
     return dict(job=job, project=project)
+
+
+async def _get_task_check_owner(
+    *,
+    task_id: int,
+    user: User,
+    db: AsyncSession,
+) -> Task:
+
+    task = await db.get(Task, task_id)
+    if not task:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Task {task_id} not found.",
+        )
+
+    if not user.is_superuser:
+        if task.owner is None:
+            raise HTTPException(
+                status_code=status.HTTP_403_FORBIDDEN,
+                detail=("Only a superuser can get a Task with `owner=None`."),
```

Comment:
I'd rather go back from "get" to "modify" here (since for the moment all users can get all tasks).